Userland/Libraries/LibGfx/ImageFormats/JPEGLoader.h

```diff
@@ -7,13 +7,26 @@
 
 #pragma once
 
+#include <AK/Array.h>
+#include <AK/HashMap.h>
 #include <AK/MemoryStream.h>
+#include <AK/Optional.h>
 #include <LibGfx/ImageFormats/ImageDecoder.h>
+#include <LibGfx/ImageFormats/JPEGHuffmanTable.h>
 
 namespace Gfx {
 
 namespace JPEG {
 struct LoadingContext;
+
+using QuantizationTables = Array<Optional<Array<u16, 64>>, 4>;
+
+struct Tables {
+    QuantizationTables quantization_tables {};
+    HashMap<u8, HuffmanTable> dc_tables {};
```

Comment:
Please use an Array for this, not a HashMap.

Reply:
Same here, I know that a `HashMap` is overkill, but that's the easiest way to write what we want.